src/Makefile

```diff
@@ -433,9 +433,9 @@ failtarget:
 help: failtarget
 	@echo
 	@echo "General options: "
-	@echo "   make V=1                     Display full compiler messages"
-	@echo "   make DEBUG=1                 Build with symbols and without optimization"
-	@echo "   make DEBUGAD                 Enables extra debugging logging in ossec-analysisd"
+	@echo "   make V=yes                   Display full compiler messages. Valid values 1, yes, YES, y and Y, in other case V is disabled"
```

Comment:
Please change `Valid values 1, yes, ...` to `Allowed values are 1, yes..., otherwise, the flag is ignored`. For all the messages in the Makefile.